Who We Are Component

mmart570 opened the pull request:
<img width="2056" alt="Screenshot 2025-05-06 at 2 42 13 PM" src="https://github.com/user-attachments/assets/790d04c3-26f0-4c7c-9e8e-8dac2b5b3561" />
Finished component, not sure if the spacing between header and paragraph should be larger

brandontran1220 reviewed (changes requested):
Move your GetInvolved.tsx to components/about

mmart570 commented:
<img width="2056" alt="Screenshot 2025-05-08 at 6 25 44 PM" src="https://github.com/user-attachments/assets/e7bdae1e-a9d7-4bc8-962c-1a4704bee7e2" />
addressed comments

brandontran1220 reviewed (changes requested):
Looks good so far! Can you make the text bigger and also move the GetInvolved.tsx file into a src/components/about. Just drag that file into there. Make it so that the text is closer like the figma. Thank you!

brandontran1220 reviewed (changes requested):
![image](https://github.com/user-attachments/assets/210de88f-74bf-460b-8b6f-37880fa312a9)

From my side, the text is hugging the wall and the text is too big.

brandontran1220 commented on src/components/home/WhoAreWe.tsx:
```diff
@@ -1,5 +1,43 @@
+          <span className="underline">Who</span> are
+          <span className="italic"> We</span>?
+        </h1>
+        <div className="flex-col px-60">
```
```suggestion
        <div className="flex-col">
```
Take out the padding and use gap instead

brandontran1220 commented on src/components/home/WhoAreWe.tsx:
```diff
@@ -1,5 +1,43 @@
+      <div className="bg-naama-ivory-100 absolute inset-0 opacity-80" />
+
+      <div className="relative z-10 flex h-full w-full items-center justify-center gap-20">
+        <h1 className="text-naama-blue-200 font-playfair text-8xl">
```
```suggestion
        <h1 className="text-naama-blue-200 font-playfair text-8xl text-center">
```

brandontran1220 reviewed (changes requested):
Mess around with the text sizing until it looks similar to the figma. And can you give me a screenshot on how it looks on your end please.

tingtingtingtingtingtingtingtingtingtin commented on src/components/home/WhoAreWe.tsx:
```diff
@@ -1,5 +1,43 @@
 const WhoAreWe = () => {
-  return <div>Who Are We?</div>;
+  return (
+    <div className="relative h-[800px] w-full">
```
```suggestion
    <div className="relative h-96 w-full">
```

avoid using px where possible

tingtingtingtingtingtingtingtingtingtin commented on src/components/home/WhoAreWe.tsx:
```diff
@@ -1,5 +1,43 @@
+      />
+      <div className="bg-naama-ivory-100 absolute inset-0 opacity-80" />
+
+      <div className="relative z-10 flex h-full w-full items-center justify-center gap-20">
```
```suggestion
      <div className="relative z-10 flex h-full w-full items-center gap-20 px-32">
```

tingtingtingtingtingtingtingtingtingtin commented on src/components/home/WhoAreWe.tsx:
```diff
@@ -1,5 +1,43 @@
+
+      <div className="relative z-10 flex h-full w-full items-center justify-center gap-20">
+        <h1 className="text-naama-blue-200 font-playfair text-8xl">
+          <span className="underline">Who</span> are
```
```suggestion
          <span className="underline decoration-2 underline-offset-8">Who</span> are
```

make the underline a little thinner/farther from the text

tingtingtingtingtingtingtingtingtingtin commented on src/components/home/WhoAreWe.tsx:
```diff
@@ -1,5 +1,43 @@
+          <span className="underline">Who</span> are
+          <span className="italic"> We</span>?
+        </h1>
+        <div className="flex-col px-60">
+          <p className="font-nunito text-naama-blue-200 max-w-5xl pb-10 text-4xl">
```
```suggestion
        <div className="flex flex-col gap-4">
          <p className="font-nunito text-naama-blue-200 p-10 text-xl">
```

flex-col requires flex first

tingtingtingtingtingtingtingtingtingtin commented on src/components/home/WhoAreWe.tsx:
```diff
@@ -1,5 +1,43 @@
+            <span className="font-extrabold"> service initiatives</span>.
+          </p>
+          <Button
+            className="flex justify-self-end"
```
```suggestion
            className="mr-14 ml-auto"
```

components shouldnt use flex unless they have children

tingtingtingtingtingtingtingtingtingtin commented on src/components/about/GetInvolved.tsx:
uncommit this file by running `git rm src/components/about/GetInvolved.tsx --cached`, then commit (without staging the file)

tingtingtingtingtingtingtingtingtingtin reviewed (changes requested):
adding to brandon's suggestions
make sure to uncommit the getinvolved change as it was already moved to /about
also remember to run `npm run format` before committing changes

mmart570 commented:
<img width="2056" alt="Screenshot 2025-05-12 at 10 36 03 PM" src="https://github.com/user-attachments/assets/38344afa-e311-4f15-8499-d41a53ff5566" />

addressed comments, didnt realize the page was way zoomed out on my end
